app/vendor/mooltipass/app.js

```diff
@@ -92,7 +97,7 @@ mooltipass.app.onMessage = function(senderId, data, callbackFunction) {
         console.log("Cancel request for reqid " + inputObject.reqid);
         
         // Cancel request only implemented in v1.1
-        if (mooltipass.util.getFirmwareFunctionalityVersionFromVersionString(mooltipass.device.version) >= "v1.1" && mooltipass.device.currentReqid == inputObject.reqid)
+        if (mooltipass.util.getFirmwareFunctionalityVersionFromVersionString(mooltipass.device.version?mooltipass.device.version:'0') >= "v1.1" && mooltipass.device.currentReqid == inputObject.reqid)
```

Comment:
I will commit with v1 if you wish, setting it to 0 made sure the extension wouldn't receive a fatal error, while we can still know it is an error (while if we send v1 we can't make sure it is)